.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "Version.Net"]
+	path = Version.Net
+	url = git@github.com:StirlingLabs/Version.Net.git
```

Comment:
Please move the submodule from the repo root to the solution directory, `/Version.Net` -> `/net/Version.Net`